### PR TITLE
Update enable-users-for-enterprise-voice-online-and-phone-system-voic…

### PR DESCRIPTION
**Pull request by pupara**
…email.md

Changed Skype For Business online connector to Teams Module

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Skype/SfbServer/skype-for-business-hybrid-solutions/plan-your-phone-system-cloud-pbx-solution/enable-users-for-enterprise-voice-online-and-phone-system-voicemail.md [hunk 1]
```diff
@@ -36,14 +36,18 @@ To enable a user for Phone System Voice and voicemail, you'll need to perform so
   
 ### To enable your users for Phone System voice and voicemail
 
-1. Before you begin, check that the Skype for Business Online Connector (Windows PowerShell module) is deployed on your Front End Servers. If it's not, you can download it from [the download center](https://www.microsoft.com/download/details.aspx?id=39366). You can find more information about using this module at [Configuring your computer for Skype for Business Online management](https://technet.microsoft.com/library/dn362839%28v=ocs.15%29.aspx).
+> [!NOTE]
+> Skype for Business Online Connector is currently part of latest Teams PowerShell Module.
```
Comment (JohanFreelancer9) [inline Skype/SfbServer/skype-for-business-hybrid-solutions/plan-your-phone-system-cloud-pbx-solution/enable-users-for-enterprise-voice-online-and-phone-system-voicemail.md:40]:
```suggestion
> Skype for Business Online Connector is currently part of the latest Teams PowerShell Module.
```

File: Skype/SfbServer/skype-for-business-hybrid-solutions/plan-your-phone-system-cloud-pbx-solution/enable-users-for-enterprise-voice-online-and-phone-system-voicemail.md [hunk 2]
```diff
@@ -36,14 +36,18 @@ To enable a user for Phone System Voice and voicemail, you'll need to perform so
...
+> If you're using the latest [Teams PowerShell public release](https://www.powershellgallery.com/packages/MicrosoftTeams/), you don't need to install the Skype for Business Online Connector.
+
+1. Before you begin, check that the Teams PowerShell module is installed on your Front End Servers. If it's not, please  install using the instructions in [Teams PowerShell Module Insallation](https://docs.microsoft.com/microsoftteams/teams-powershell-install).
```
Comment (JohanFreelancer9) [inline Skype/SfbServer/skype-for-business-hybrid-solutions/plan-your-phone-system-cloud-pbx-solution/enable-users-for-enterprise-voice-online-and-phone-system-voicemail.md:43]:
```suggestion
1. Before you begin, check that the Teams PowerShell module is installed on your Front End Servers. If it's not, please install it using the instructions in [Teams PowerShell Module Insallation](https://docs.microsoft.com/microsoftteams/teams-powershell-install).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by scanum** [comment]
@CarolynRowe PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**